src/main/resources/db/changelog/changes/v1.1.0/update-drv-purchase-order-line-details-definition.xml

```diff
@@ -246,11 +246,11 @@
               "visibleByDefault": true
             },
             {
-              "name": "purchase_order_notes",
+              "name": "PO notes",
```

Comment:
These should keep the underscore format, so like "po_notes". And then the backend name will have to be changed in the translations files. As it is right now, I don't think the translations will work properly (because there's no "PO notes" key in the translations file)